### DISCUSSION TITLE
Onboard LED not working

### DISCUSSION THREAD

**Discussion by dscholtenemis** [jomjol/AI-on-the-edge-device#3181] [Q&A] [answered]
I have two systems running (hot water and cold water) cold water system is running perfect.
During warm water setup i clicked on "Live stream with LED on", but the LED stays off.
I am using the onboard flash LED. I tried the power supply and USB Cable from the working cold water system, same result, LED stays off.

**Comment by SybexX** [comment]
I would say that if both systems are running with the same firmware and the same settings, your onboard flash LED will probably be broken.

**Reply by dscholtenemis** [reply]
I thought the same.
Sorry for not meantion it. 
I bought two new devices, same problem. :-(

**Reply by SybexX** [reply]
Have you ever measured the voltage on the GPIO4 or connected an external LED to the GPIO4 for testing?

![Unbenannt](https://github.com/user-attachments/assets/c345fac6-6a5d-491f-a63b-f56a965cd2c9)

**Reply by caco3** [reply]
> I bought two new devices, same problem. :-(

What does this mean?
Again one is ok and one is bad?

**Reply by dscholtenemis** [reply]
First, I bought 2 circuit boards, and the first one works without any problems.
On the second one, the camera flash LED does not work.
I thought it was a hardware defect and bought 2 more circuit boards. On both of them, the camera flash LED does not work.
As soon as I open the livestream page with the camera flash LED ON, I can measure 1.62V at GPIO4.
Powersupply is 5.0V.

**Reply by SybexX** [reply]
![358911677-b2af3a74-8d26-480f-8caa-d5eb2c0223ef](https://github.com/user-attachments/assets/6e3a8b50-4a5d-484f-ab7b-fa037190a206)

 
![358422291-c345fac6-6a5d-491f-a63b-f56a965cd2c9](https://github.com/user-attachments/assets/e8ac0f8c-7b1d-433f-a5e1-c2492c18d99e)

**Reply by dscholtenemis** [reply]
I did all measurements.
With LED on or OFF I have the same results.
GPIO4 to 3.3V -> 3.3V 
GPIO4 to GND -> 0V 

For S8050 same results.
Always 0V.

I can't believe that I have 3 new non-functional boards. :-( 
Is there anything else that I can check on configuration?

**Reply by SybexX** [reply]
![Unbenannt](https://github.com/user-attachments/assets/f3777114-80f1-41d1-9b3b-6f91b864d924)

**Reply by dscholtenemis** [reply]
On both systems it looks like this:
![image](https://github.com/user-attachments/assets/66ddf498-3c8a-45db-b0d5-9b3cbd9447b6)
as soon i check the box it looks like this:
![image](https://github.com/user-attachments/assets/e097965e-3b96-4547-b2e5-98da8b65d084)

**Reply by SybexX** [reply, marked as answer]
if the onboard flash led is used, the option "GPIO" should not be activated.

![esp32-s-ai_pinout_zc](https://github.com/user-attachments/assets/ad07493b-6544-4ea6-88ec-5ec5dca3c5ce)

You could try to measure directly on the ESP32 module to rule out a defect in the conductor track.

**Reply by dscholtenemis** [reply]
All 3 new boards have the same issue :-(
![image](https://github.com/user-attachments/assets/0d7c561b-003e-45af-838f-4afc6fa543c1)

**Reply by caco3** [reply]
What does the circle mean?
And the soldered wire?

**Reply by SybexX** [reply]
The conductor track is probably broken and he bridged it with the cable.
